query/src/test/scala/filodb/query/PlanValidationSpec.scala

```diff
@@ -17,6 +17,8 @@ trait PlanValidationSpec extends Matchers with StrictLogging {
     planString
       .replaceAll("testProbe-.*]", "testActor]")
       .replaceAll("InProcessPlanDispatcher.*\\)", "InProcessPlanDispatcher")
+      .replaceAllLiterally("['", "")
+      .replaceAllLiterally("\n", "")
```

Comment:
Can these be removed? These plans should be more-or-less identical to e.g. `count_over_time` (and it looks like that holds true for the new test case).

Reply:
It is not and that is the reason I had to add this to get the tests passed and I ran the whole tests to make sure others are not broken.

Reply:
It looks like the plan you copied is the `shouldEqual` output (which contains `[` and `]` characters to indicate the difference between expected/actual strings). There are two strings each failed test outputs:
```
~~~~~~~~~~~~~~~~ EXPECTED ~~~~~~~~~~~~~~~~
...
~~~~~~~~~~~~~~~~ ACTUAL ~~~~~~~~~~~~~~~~
...
```

Could you copy the `ACTUAL` string into the test? That should let you remove these two new `replaceAllLiterally`s.

Reply:
Just to add, please don't literally copy but make sure the plan looks good.